migration/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "migration",
   "version": "1.0.0",
+  "type": "module",
```

Comment:
why was this added?

Reply:
app.ts is recognised as an es module and so importing it into index.ts doesnt work with commonjs, docker loses its mind without the module type definition. Perhaps you have a different solution?
<img width="587" alt="Screenshot 2023-07-16 at 10 46 36 am" src="https://github.com/csesoc/cselectives-v2/assets/76886689/69d53aeb-ec5b-4777-bcdf-20d3e5344898">

Reply:
okies congratulations or sorry to hear that